releasenotes/notes/0.5/0_5_release-89f59845afb19e89.yaml

```diff
@@ -0,0 +1,13 @@
+---
+prelude: >
+    The Qiskit Experiments 0.5 release added the visualization module for
+    drawing figures which replaces the previous plotting functionality. The
+    speed of randomized benchmarking experiments has been significantly
+    improved. The ``qubit`` and ``qubits`` input to experiments has been
+    regularized to ``physical_qubits``, and ``qiskit-ibmq-provider`` has been
+    deprecated in favor of ``qiskit-ibm-provider``. A new
+    :class:`.MultiStateDiscrimination` experiment has been added for classifying
+    IQ data. A test backend that uses Qiskit Dynamics for pulse-level
```

Comment:
I would mention PulseBackend so the user can find more information. Though do we expect users to use this outside of qiskit-experiments tests? I wonder if it should be called out as a highlight.

Reply:
That's fair, I removed it.